css/index.css

```diff
@@ -183,21 +175,30 @@ table#tableId tr:nth-child(odd) {
   background-color: #eee;
 }
 
-#bookListm div p {
-  display: inline-block;
+#bookList div {
+  display: flex;
+  justify-content: space-between;
 }
 
-/* hide and show elements */
-.show-color {
-  color: rgb(248, 249, 252);
-  background-color: rgb(212, 40, 40);
+#bookList div p {
+  display: inline-block;
+  margin: 20px;
 }
 
 .hide-nav {
   display: none;
 }
 
 input {
-  width: 30vw;
+  display: flex;
+  width: 50vw;
   height: 35px;
 }
+
+.remove {
+  height: fit-content;
+  margin: 20px;
+  background-color: red;
+  color: #fff;

```

Comment:
- [Optional] I would advise you to add a CSS property of `cursor: pointer` on your buttons (like the Add and delete button) it will improve the user experience.